.circleci/config.yml

```diff
@@ -270,7 +270,7 @@ workflows:
   run-tests:
     jobs:
       - Rust tests - stable
-      - Rust tests - beta
+      #- Rust tests - beta
```

Comment:
Can you file a bug about adding them back and drop the bug number, as a comment, here?